### ISSUE TITLE
[BUG] - New Message Notification

### ISSUE THREAD

**Issue by AA303** [nanu-c/axolotl#281]
### Description

New message notification is missing when you lock Axolots device is inside a conversation. 

### Steps to Reproduce

1. Open a conversation and lock the device
2. Ask sender to send a test message
3. No audio notification will be triggered 

**Expected behavior:** [What you expect to happen]

You would expect the same audio notification as if you exit the chat and lock the device on the list of conversations.

**Actual behavior:** [What actually happens]

Audio notification is not triggered when device is locked while inside Axolotl conversation.

#### Versions

0.9.0 and earlier

Please provide the Version as written in `Settings->Advanced-> Submit Debug Log`

#### Device

Nexus 5

#### Link to Debug Log
Please provide a link to debug

**Comment by Flaburgan** [comment]
I agree this is annoying, I'm often missing messages because of that. I'm tagging it for 0.9.3 at the moment, and we'll see if we can fix it.
